GenderPayGap.Core/Extensions/AspNetCore/Config.cs

```diff
@@ -71,101 +68,38 @@ public static string EnvironmentName
         }
 
 
-        public static string SiteAuthority
-        {
-            get
-            {
-                string siteName = GetAppSetting("EXTERNAL_HOST");
-                return $"https://{siteName}/";
-            }
-        }
-
-        public static IConfiguration Build(Dictionary<string, string> additionalSettings = null, IConfigurationBuilder builder = null)
+        public static IConfiguration Build(IConfigurationBuilder builder = null)
         {
             builder = builder ?? new ConfigurationBuilder();
 
-            builder.AddJsonFile("appsettings.json", false, true);
-            builder.AddJsonFile($"appsettings.{EnvironmentName}.json", true, true);
-
-            builder.AddEnvironmentVariables();

```

Comment:
This `builder.AddEnvironmentVariables();` is a duplicate - we now just have it once at the end